### PR TITLE
Java package naming

### LINKED ISSUE #105: Add java package naming to naming conventions
## What
Clarify what our Java package names should be

## Why
To clarify standard for developers & testers

### PR DESCRIPTION
**Pull request by yantantether**
fixes #105

### CODE REVIEW

File: src/development/coding-naming-conventions.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Different case standards should be applied depending on usage:
 
 * `Natural` - Using natural language standard, with spaces to separate words and capitals for initial letter and proper nouns
 * `kebab-case` - All lower case with dashes to separate words
-* `CamelCase` - Initial letters in words are uppercased, with no separators
+* `CamelCase` - Initial letters in words are uppercased, with no separators. First letter may be upper or lower.
```
Comment (LouisRoberts) [inline src/development/coding-naming-conventions.md:114]: Further info on camelCase at https://www.techtarget.com/whatis/definition/CamelCase#:~:text=What%20is%20CamelCase%3F,humps%20on%20a%20camel's%20back.
I always use first character lower case - but if you can uppercase it I suppose consistency is the key. Java tends to be lowerCamelCase.

File: src/development/coding-naming-conventions.md [hunk 2]
```diff
@@ -165,6 +165,16 @@ Merge/pull requests should be raised with `natural` case and this convention:
 
 For language constructs refer to the [standard language style guides](../coding-style-guide/).
 
+::: card
+
+### Java
+
+Java `packages` should be all lower case concatenated with this convention:
+
+> `uk.nhs.nhsbsa.{service}`
```
Comment (LouisRoberts) [inline src/development/coding-naming-conventions.md:174]: What about common libraries ?

File: src/development/coding-naming-conventions.md (hunk 1 above)
Comment (yantantether) [reply src/development/coding-naming-conventions.md:114]: It depends on the circumstance @LouisRoberts. Class names are `CamelCase`, method names are `camelCase`

File: src/development/coding-naming-conventions.md (hunk 2 above)
Comment (yantantether) [reply src/development/coding-naming-conventions.md]: Updated

**Review by LouisRoberts — APPROVED** [review]

**Review by LouisRoberts — APPROVED** [review]
Accepted developer responses and updates

**Review by LouisRoberts — APPROVED** [review]

**Pull request merged**